docs/deploy-models-using-seldon.md

```diff
@@ -0,0 +1,52 @@
+# Deploy models on OpenShift using Seldon Server
+
+Have a model trained and ready to share with the world? In this guide, we go over how to deploy models as a microservice using one of Seldon Core’s pre-packaged [model servers](https://docs.seldon.io/projects/seldon-core/en/v1.1.0/servers/overview.html).
+
+## Pre-requisites
+
+* An ML model trained using one of these supported ML [libraries](https://docs.seldon.io/projects/seldon-core/en/latest/nav/config/servers.html) or a custom model (guide forthcoming) .
+* OpenShift namespace with [seldon operator installed](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/install.html#openshift) and administrator privileges available.
+
+## Steps to deploy
+
+1. Save the model as a .joblib file and name it `model.joblib`. Push the model to S3 in its own directory. For example a path like `s3://<bucket_name>/<project_name>/<model_name>/model.joblib`.
+
+2. On the OpenShift console:
+
+    * Ensure the Seldon-Core operator is installed in the namespace where you are deploying.
+        * If you are using the [Operate First cluster](https://console-openshift-console.apps.zero.massopen.cloud/), you can use the namespace `ds-ml-workflows-ws` which has Seldon deployed.
+        * To get access to this namespace, [open an issue](https://github.com/operate-first/support/issues/new/choose) on the operate-first/support repository.
+
+    * In the namespace, add create a secret with the AWS credentials for access to the bucket. To create a Secret, go to `Workloads`, `Secrets`, click on `Create` and select `From YAML` in the dropdown menu. 
+  
+    ![create secret](../public/assets/deploy-models-using-seldon/create-secret.png)
+
+    Here is an example of the format of the secret:
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+    name: seldon-init-container-secret
+    type: Opaque
+    data:
+    AWS_ACCESS_KEY_ID: XXXX
+    AWS_SECRET_ACCESS_KEY: XXXX
+    AWS_ENDPOINT_URL: XXXX
+    USE_SSL: XXXX
+    ```
+    Find documentation on how to configure different object storage providers [here](https://docs.seldon.io/projects/seldon-core/en/latest/servers/kfserving-storage-initializer.html).
```

Comment:
should this be italicized? feels like a note referencing the above, but given the block of code and following instruction seems confusing as to what is is referring to.